src/TestInfrastructure/app.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
.config file are not used throughout the project. They are generated (each time) when refreshing nuget packages. We keep them to avoid undoing changes in multiple places each time we refresh.